content/en/tracing/trace_collection/trace_context_propagation/_index.md

```diff
@@ -43,8 +44,10 @@ Use the following environment variables to configure formats for reading and wri
 
 `DD_TRACE_PROPAGATION_STYLE`
 : Specifies trace context propagation formats for extraction and injection in a comma-separated list. May be overridden by extract-specific or inject-specific configurations.<br>
-**Default**: `datadog,tracecontext` <br>
-**Note**: With multiple formats, extraction follows the specified order (for example, `datadog,tracecontext` checks Datadog headers first). The first valid context continues the trace; additional valid contexts become span links.
+**Default**: `datadog,tracecontext,baggage` <br>
```

Comment:
I don't think we should change this yet since it's not true for any of the tracers.

Reply:
Just the default and note below it right?

Reply:
Or do you mean revert the entire file

Reply:
The change in line 32 above is also "By default, the Datadog SDK extracts and injects distributed tracing headers using the following formats," which is not available yet. 😬

Reply:
I would say, don't add anything regarding configuration just yet. `:2cents:`